XmlFormatterOsIndependent/Factories/UpdateManagerFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using XmlFormatterModel.Update;
+using XmlFormatterModel.Update.Strategies;
+
+namespace XmlFormatterOsIndependent.Update
+{
+    /// <summary>
+    /// This class will create you a valid update manager
+    /// </summary>
+    public class UpdateManagerFactory : IVersionManagerFactory
+    {
+        /// <summary>
+        /// The version manager
+        /// </summary>
+        private IVersionManager manager;
+
+        /// <summary>
+        /// Create a new instance of this class
+        /// </summary>
+        public UpdateManagerFactory()
+        {
+            manager = null;
+        }
+
+        /// <summary>
+        /// Get the version manager to use
+        /// </summary>
+        /// <returns>The correct version manager</returns>
+        public IVersionManager GetVersionManager()
+        {
+            if (manager == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '??=' operator here.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)